crates/router/src/connector/aci/transformers.rs

```diff
@@ -91,12 +91,154 @@ impl TryFrom<&api_models::payments::WalletData> for PaymentDetails {
                     account_id: None,
                 }))
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
+            api_models::payments::WalletData::MbWayRedirect( _ ) => {
+                Err(errors::ConnectorError::NotImplemented(
+                    "Payment method".to_string(),
+                ))?
+            }
+            api_models::payments::WalletData::AliPayRedirect( _ ) => {
+                Err(errors::ConnectorError::NotImplemented(
+                    "Payment method".to_string(),
+                ))?
+            }
```

Comment:
```suggestion
            api_models::payments::WalletData::MbWayRedirect( _ ) |
            api_models::payments::WalletData::AliPayRedirect( _ ) => {
                Err(errors::ConnectorError::NotImplemented(
                    "Payment method".to_string(),
                ))?
            }
```
Please use OR for the rest of the match cases. Please refer to the above example